From 60c689bca268495181264d074941369ff1457af9 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:36:54 -0400
Subject: [alsa] hda: Remove superfluous MAX_AZX_DEV

Message-id: <140972823573-360-patchset-jkysela@redhat.com>
Patchwork-id: 91056
O-Subject: [RHEL 7.1 PATCH 359/442] ALSA: hda - Remove superfluous MAX_AZX_DEV
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

MAX_AZX_DEV is no longer referred anywhere, let's kill it.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 085ec0d945cd52caba3765e4cbcfe24cc136e905)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_priv.h b/sound/pci/hda/hda_priv.h
index 4a7cb01..da12d10 100644
--- a/sound/pci/hda/hda_priv.h
+++ b/sound/pci/hda/hda_priv.h
@@ -120,9 +120,6 @@ enum { SDI0, SDI1, SDI2, SDI3, SDO0, SDO1, SDO2, SDO3 };
 #define TERA_NUM_CAPTURE 3
 #define TERA_NUM_PLAYBACK 4
 
-/* this number is statically defined for simplicity */
-#define MAX_AZX_DEV  16
-
 /* max number of fragments - we may use more if allocating more pages for BDL */
 #define BDL_SIZE  4096
 #define AZX_MAX_BDL_ENTRIES (BDL_SIZE / 16)
-- 
1.7.1